projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -89,25 +89,60 @@
 				}
 			} );
 
-			makeAjaxRequest(
-				'GET',
-				ajaxurl + '?action=upsell_nudge_jitm&_ajax_nonce=' + jetpackAdminMenu.upsellNudgeJitm,
-				undefined,
-				null,
-				function ( xhr ) {
-					try {
-						if ( xhr.readyState === XMLHttpRequest.DONE ) {
-							if ( xhr.status === 200 && xhr.responseText ) {
-								adminMenu
-									.querySelector( '#toplevel_page_site_card' )
-									.insertAdjacentHTML( 'afterend', xhr.responseText );
+			const siteUrl = new URL( jetpackAdminMenu.siteUrlOption );
+			let siteHostname = null;
+
+			if ( 'hostname' in siteUrl ) {
+				siteHostname = siteUrl.hostname;
+			}
+
+			if ( jetpackAdminMenu.launchpadScreenOption === 'full' ) {
+				const LaunchpadNotice =
```

Comment:
Note: Hard coding the notice markdown is what gives me the most pause in this PR ( issues with maintenance, hard coding, etc. ). Wondering if there are smarter ways of handling this. 🤔

Reply:
A couple of notes on hardcoded markup:

1. It looks gross, but more gross would be loading all of React to give us what we're used to. 
2. You can use template literals (aka backticks <code>`</code>) to avoid having to concatenate all of your string stuff - they respect newlines.

Reply:
Super helpful advice. I wasn't sure if there was some clever way of loading the shared components that already exist for this notice. Thanks @mattwiebe